src/shared/lib/StoreUtils.spec.ts

```diff
@@ -288,10 +288,20 @@ describe('StoreUtils', () => {
         });
     });
 
-    it("won't fetch onkokb data if there are no mutations", (done) => {
-        fetchOncoKbData({}, emptyMutationData).then((data: any) => {
-            assert.deepEqual(data, {sampleToTumorMap: {}, indicatorMap: {}});
-            done();
+    describe('fetchOncoKbData', () => {
+        it("won't fetch onkokb data if there are no mutations", (done) => {
+            fetchOncoKbData({}, emptyMutationData).then((data: any) => {
```

Comment:
interesting that you have to use done.  i guess js promises are always async even then there is no async call inside.